src/sos_polynomial_in_semialgebraic_set_bridge.jl

```diff
@@ -9,10 +9,9 @@ function lagrangian_multiplier(model::MOI.ModelLike, p, set::SOSLikeCone, q,
     # If maxdegree_s2 is odd, div(maxdegree_s2,2) would make s^2 have degree up to maxdegree_s2-1
     # for this reason, we take div(maxdegree_s2+1,2) so that s^2 have degree up to maxdegree_s2+1
     maxdegree_s = div(maxdegree_s2 + 1, 2)
-    # FIXME handle the case where `p`, `q_i`, ...  do not have the same variables
-    # so instead of `variable(p)` we would have the union of them all
-    @assert variables(q) ⊆ variables(p)
-    monos = MP.monomials(MP.variables(p), mindegree_s:maxdegree_s)
+    vars = sort([MP.variables(p)..., MP.variables(q)...])
```

Comment:
`sort!` is more efficient as it will do it in-place instead of creating a new array